lib/db-connection-info.js

```diff
@@ -0,0 +1,19 @@
+
+const user = process.env.POSTGRES_USER;
+const password = process.env.POSTGRES_PASSWORD;
+
+const host = process.env.POSTGRES_HOST;
+const database = process.env.POSTGRES_DB;
+const port = process.env.POSTGRES_PORT;
+const max = process.env.POSTGRES_POOL;
+const idleTimeoutMillis = process.env.POSTGRES_TIMEOUT;
+
+module.exports = {
+  user,
```

Comment:
Since this is the only place `user` is used and that constant isn't exported, why not remove the `const` line and write this line like this?
````
  user: process.env.POSTGRES_USER,
````
Ditto for the other properties.